__tests__/browser.test.js

```diff
@@ -0,0 +1,104 @@
+const puppeteer = require('puppeteer');
+
+describe('Google Search Test', () => {
+    let page, browser;
+    const option = {
+        headless: false,
+        slowMo: 150,
+        args: [
+            '--window-size=1600,950',
+            '--window-position=100,50',
+            '--no-sandbox'
+        ]
+    };
+
+    beforeEach (async () => {
+        browser = await puppeteer.launch();
+        page = await browser.newPage();
+        await page.goto('https://www.google.co.jp');
+    });
+
+    afterEach (async () => {
+        await browser.close();
+    }); 
+
+    test('should visit Google', async () => {
+        expect(await page.title()).toEqual('Google');
+    });
+
+    test('should have search bar', async () => {
+        expect(await page.waitFor('input[name=q]'))
+        .toBeCalled;
```

Comment:
@tyamadaaa 
1. `.toBeCalled` is a `Function` so you need to call it as a function, meaning `.toBeCalled();`
2. You're not actually testing anything here. This test should look like this:
```
        await page.waitFor('input[name=q]')
        expect(await page.evaluate(() => document.querySelectorAll('input[name=q]').length))
        .toEqual(1);
```